draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1581,75 +1581,121 @@ Steps:
 26. return (x, y)
 ~~~
 
-## Mappings for twisted Edwards curves
+## Mappings for Edwards curves
 
-### Elligator 2 Method {#ell2edwards}
+Edwards curves are closely related to Montgomery
+curves ({{montgomery}}): every Edwards curve is birationally equivalent
+to a Montgomery curve ({{BBJLP08}}, Theorem 3.2).
+This equivalence yields an efficient way of hashing to an Edwards curve:
+first, hash to the equivalent Montgomery curve, then transform the
+result into a point on the Edwards curve by evaluating a pair of rational
+functions called a birational map.
+Thus, hashing to an Edwards curve requires first identifying a
+corresponding Montgomery curve and birational map.
+
+### Birational maps from Montgomery to Edwards curves {#birational-map}
+
+There are two ways to identify the Montgomery curve and birational map
+for use with a given Edwards curve.
 
-Twisted Edwards curves and Montgomery curves are closely related: every
-twisted Edwards curve is birationally equivalent to a Montgomery curve
-({{BBJLP08}}, Theorem 3.2). To hash to a twisted Edwards curve, hash
-to the equivalent Montgomery curve and evaluate the rational map to
-obtain a point on the twisted Edwards curve.
+When hashing to a standard Edwards curves for which a corresponding
+Montgomery form and birational map are also standardized, the standard
+Montgomery form and birational map MUST be used to ensure compatibility
+with existing software.
+Two examples of this are the edwards25519 and edwards448 curves,
+which correspond to the Montgomery curves curve25519 and curve448, respectively.
+For both of these curves, {{RFC7748}} lists both the Montgomery and Edwards
+forms and gives the corresponding rational maps.
```

Comment:
Should we also specify that new Edwards curves SHOULD/MUST specify a corresponding Montgomery form and birational map?

Reply:
Not a bad idea, though I wonder if people might complain that trying to require this would be mission creep for this standard...

(But I suppose that's what SHOULD is for!)

Reply:
Yeah, I'm leaning towards SHOULD. :-)